lib/src/main/java/com/auth0/jwt/creators/AccessJwtCreator.java

```diff
@@ -281,8 +279,8 @@ public String signBase32Encoding(Algorithm algorithm) throws Exception {
      * @throws Exception if all the standard claims weren't provided
      */
     private void verifyClaims() throws Exception {
-        for(String claim : addedClaims.keySet())
-            if(!addedClaims.get(claim))
+        for(String claim : requiredClaims.keySet())
+            if(!requiredClaims.get(claim))
                 throw new Exception("Standard claim: " + claim + " has not been set");
```

Comment:
Can we please throw something more descriptive than "Exception"? i.e. `IllegalStateException/IllegalArgumentException` or even a `RequiredClaimException`

Reply:
done